lively.ide/window-switcher.cp.js

```diff
@@ -92,6 +92,7 @@ const WindowPreview = component({
     extent: pt(100, 100),
     fill: Color.transparent
   },
+  { name: 'overlay', fill: Color.transparent, extent: pt(130, 130), nativeCursor: 'pointer' },
```

Comment:
Why did you opt to insert a new morph for this? Wouldn't it be enough to just set the `nativeCursor` on `'window preview'` above? :thinking:

Reply:
That would work, if the previews themselves were plain textures. But they are html, and therefore happily carry over their respective css cursor styles, so the easiest way to get rid of this is to just slap a layer on top.

Reply:
Aaah, yes! That makes sense!